analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -578,6 +578,16 @@ class Pub(
 
     private fun readPackageInfoFromCache(packageInfo: JsonNode): JsonNode {
         val definitionFile = reader.findFile(packageInfo, "pubspec.yaml")
+        if (definitionFile == null) {
+            createAndLogIssue(
+                source = managerName,
+                message = "Could not find 'pubspec.yaml' for '${packageInfo["name"].textValueOrEmpty()}'.",
```

Comment:
Does it make sense to move `"pubspec.yaml"` to a variable now, and also use it above?

Reply:
How about a `const` as this string is being used multiple times in this class?

Reply:
> How about a `const` as this string is being used multiple times in this class?

:+1:

Reply:
Now you still need to use that constant ;-)

Reply:
Done. Sorry for that :)